Lib/test/test_email/test_utils.py

```diff
@@ -145,10 +146,9 @@ def test_localtime_epoch_notz_daylight_false(self):
     # XXX: Need a more robust test for Olson's tzdata
```

Comment:
And perhaps these three lines could be removed?

Reply:
done, thx.